onnx/helper.py

```diff
@@ -171,6 +172,24 @@ def make_opsetid(domain, version):  # type: (Text, int) -> OperatorSetIdProto
     return opsetid
 
 
+def make_function(
+    domain,  # type: Text
+    fname,  # type: Text
+    inputs,  # type: List[Text]
+    outputs,  # type: List[Text]
+    nodes,  # type: List[NodeProto]
+    opset_imports,    # type: Sequence[OperatorSetIdProto]
```

Comment:
Suggest adding other fields in FunctionProto, with default-values? Default empty list may make sense for opset_imports also.
```python
   attributes = [],
   doc_string = ''
```